cwms-data-api/src/main/java/cwms/cda/api/watersupply/WaterUserController.java

```diff
@@ -0,0 +1,117 @@
+import io.javalin.plugin.openapi.annotations.HttpMethod;
+import io.javalin.plugin.openapi.annotations.OpenApi;
+import io.javalin.plugin.openapi.annotations.OpenApiContent;
+import io.javalin.plugin.openapi.annotations.OpenApiParam;
+import io.javalin.plugin.openapi.annotations.OpenApiResponse;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.servlet.http.HttpServletResponse;
+import org.jetbrains.annotations.NotNull;
+import org.jooq.DSLContext;
+
+
+public final class WaterUserController extends WaterSupplyControllerBase implements Handler {
+    private static final Logger LOGGER = Logger.getLogger(WaterUserController.class.getName());
+
+    public WaterUserController(MetricRegistry metrics) {
+        waterMetrics(metrics);
+    }
+
+    @OpenApi(
+        pathParams = {
+            @OpenApiParam(name = OFFICE, description = "The office Id the contract is associated with.",
+                    required = true),
+            @OpenApiParam(name = PROJECT_ID, description = "The project Id the contract is associated with.",
+                    required = true),
+            @OpenApiParam(name = WATER_USER, description = "The water user the contract is associated with.",
+                    required = true)
+        },
+        responses = {
+            @OpenApiResponse(status = STATUS_200,
+                content = {
+                    @OpenApiContent(type = Formats.JSONV1, from = WaterUserContract.class)
+                })
+        },
+        description = "Gets a specified water user.",
+        method = HttpMethod.GET,
+        path = "/projects/{office}/{project-id}/water-user/{water-user}",
+        tags = {TAG}
+    )
+
+    @Override
+    public void handle(@NotNull Context ctx) {
+        try (Timer.Context ignored = markAndTime(GET_ONE)) {
+            String location = ctx.pathParam(PROJECT_ID);
+            String office = ctx.pathParam(OFFICE);
+            CwmsId projectLocation = buildCwmsId(office, location);
+            DSLContext dsl = getDslContext(ctx);
+            String entityName = ctx.pathParam(WATER_USER);
+            String formatHeader = ctx.header(Header.ACCEPT) != null ? ctx.header(Header.ACCEPT) : Formats.JSONV1;
+            ContentType contentType = Formats.parseHeader(formatHeader, WaterUserContract.class);
+            ctx.contentType(contentType.toString());
+            WaterContractDao contractDao = getContractDao(dsl);
+            WaterUser user = contractDao.getWaterUser(projectLocation, entityName);
+
+            if (user == null) {
+                CdaError error = new CdaError("No water user found for the provided parameters.");
+                LOGGER.log(Level.SEVERE, "Error retrieving water user.");
+                ctx.status(HttpServletResponse.SC_NOT_FOUND).json(error);
+                return;
+            }
+            List<WaterUser> params = new ArrayList<>();
+            params.add(user);
```

Comment:
why is this being sent as a list?

Reply:
Replaced with format call that does not require list input